atlas/AtlasMl/atlasml/models/competency.py

```
@@ -28,16 +29,42 @@ class CompetencyRelation(BaseModel):
-    competencies: List[GenerateCompetencyRequest]
+    competencies: list[GenerateCompetencyRequest]
+
 
 class GenerateCompetencyResponse(BaseModel):
-    competencies: List[Competency]
-    competency_relations: List[CompetencyRelation]
+    competencies: list[Competency]
+    competency_relations: list[CompetencyRelation]
+
 
 class GenerateEmbedingsResponse(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)